okdc-centos.sh

```diff
@@ -150,6 +164,9 @@ install_network() {
 				install_calico_with_etcd
 				;;
 			3)
+				install_calico_with_kdd
```

Comment:
Oh, no. There's another case switch just above this one which runs when $Network is predefined or in NOINPUT mode. We will need a calico_kdd (or something like that) option added so users can use that in NOINPUT mode.